### ISSUE TITLE
MVM: Skycells with negative Y values cause MVM processing to crash

### ISSUE THREAD

**Issue by stscijgbot-hstdp** [spacetelescope/drizzlepac#1063]
_Issue [HLA-584](https://jira.stsci.edu/browse/HLA-584) was created on JIRA by [Michael Dulude](https://jira.stsci.edu/secure/ViewProfile.jspa?name=dulude):_

Discovery dataset: idix03 (WFC3__15137_03)
 Skycells associated with this dataset by cell_utils.get_sky_cells() are as follows:
 * skycell-p1894x14y17
 * skycell-p1894x14y18
 * skycell-p1894x15y17
 * skycell-p1894x15y18
 * **skycell-p1974x09y-1**
 * skycell-p1974x09y00
 * skycell-p1974x09y01

MVM processing is successful for all skycells except skycell-p1974x09y-1. This is the traceback for the failure:
```java
$ runmultihap idix03_mvm.out
Traceback (most recent call last):
  File "/Users/dulude/Documents/Code/HLATransition/drizzlepac/drizzlepac/hapmultisequencer.py", line 213, in run_mvm_processing
    layer_method='all')
  File "/Users/dulude/Documents/Code/HLATransition/drizzlepac/drizzlepac/haputils/poller_utils.py", line 281, in interpret_mvm_input
    poller_type='mvm')
  File "/Users/dulude/Documents/Code/HLATransition/drizzlepac/drizzlepac/haputils/poller_utils.py", line 1005, in build_poller_table
    skycell_obj = cell_utils.SkyCell.from_name(pipeline_skycell_id)
  File "/Users/dulude/Documents/Code/HLATransition/drizzlepac/drizzlepac/haputils/cell_utils.py", line 1002, in from_name
    y = int(scell_id[9:11])
ValueError: invalid literal for int() with base 10: ''
2021189122911 INFO src=drizzlepac.hapmultisequencer- Processing completed at 2021-07-08 12:29:11.816544
2021189122911 INFO src=drizzlepac.hapmultisequencer- Total processing time: 0.088756 sec
2021189122911 INFO src=drizzlepac.hapmultisequencer- Return exit code for use by calling Condor/OWL workflow code: 0 (zero) for success, 1 for error 
2021189122911 INFO src=drizzlepac.hapmultisequencer- Return condition 1
Return Value:  1
```

**Comment by stscijgbot-hstdp** [comment]
_Comment by [Warren Hack](https://jira.stsci.edu/secure/ViewProfile.jspa?name=hack) on [JIRA](https://jira.stsci.edu/browse/HLA-584?focusedCommentId=538233#comment-538233):_

[Drizzlepac PR #&#x2060;1065](https://github.com/spacetelescope/drizzlepac/pull/1065) has been submitted to correct the logic problem identified by this dataset.  In addition, the logic was modified to insure that the X and Y values for the SkyCell are 1-based as well, as expected by the rest of the code.  

The footprints generated using SkyFootprint with each of the SkyCell's identified using 'cell_utils/get_sky_cells.py'  (see above) show that the indexing worked correctly with that PR.

**Comment by stsci-hack** [comment]
Closed by #1065.
